### ISSUE TITLE
watch with verbose and focus is now too verbose

### ISSUE THREAD

**Issue by plastikfan** [onsi/gomega#608]
I recently performed an update of the cli, from a really aged version (2.1) to the current 2.3. In the old version I could run watch with -v flag and run a single test with focus and the only output I got was anything to do with the focused test. However, with the latest release, when I run the same scenario, I get a huge amount of output listing all the tests that were skipped. This is less than ideal because now I have to scroll back through the terminal hunting for the output relating to the focused test amongst a sea of noise regarding the skipped tests.

The following is an example of the noisy output (some of the output has been skipped as it is too monotonous and not relevant):

```
S [SKIPPED]
FilterGlob Filter Children (glob) 🧪 ===> given: 'folder(with files): glob filter (negate)'
/home/plastikfan/dev/github/go/snivilised/extendio/xfs/nav/filter-glob_test.go:216
------------------------------
S [SKIPPED]
TraverseNavigator errors given: root is not a folder 🧪 ===> ('folders') should return error
/home/plastikfan/dev/github/go/snivilised/extendio/xfs/nav/traverse-navigator-error_test.go:175
------------------------------
S [SKIPPED]
TraverseNavigator Navigator Ensure Callback Invoked Once 🧪 ===> given: 'files: Path is leaf'
/home/plastikfan/dev/github/go/snivilised/extendio/xfs/nav/traverse-navigator_test.go:167
------------------------------
S [SKIPPED]
TraverseNavigator errors given: root is not a folder 🧪 ===> ('files') should return error
/home/plastikfan/dev/github/go/snivilised/extendio/xfs/nav/traverse-navigator-error_test.go:176
------------------------------
S [SKIPPED]
TraverseNavigator Navigator Ensure Callback Invoked Once 🧪 ===> given: 'files: Path contains folders'
/home/plastikfan/dev/github/go/snivilised/extendio/xfs/nav/traverse-navigator_test.go:174
------------------------------
S [SKIPPED]
TraverseNavigator errors top level QueryStatus given: error occurs 🧪 should: halt traversal
/home/plastikfan/dev/github/go/snivilised/extendio/xfs/nav/traverse-navigator-error_test.go:181
------------------------------
S [SKIPPED]
TraverseNavigator Navigator Ensure Callback Invoked Once 🧪 ===> given: 'files: Path contains folders'
/home/plastikfan/dev/github/go/snivilised/extendio/xfs/nav/traverse-navigator_test.go:181
------------------------------
S [SKIPPED]
TraverseNavigator Navigator Ensure Callback Invoked Once 🧪 ===> given: 'files: Path contains folders (large)'
/home/plastikfan/dev/github/go/snivilised/extendio/xfs/nav/traverse-navigator_test.go:189
------------------------------
S [SKIPPED]
TraverseNavigator Navigator Ensure Callback Invoked Once 🧪 ===> given: 'files: Path contains folders (large, ensure single invoke)'
/home/plastikfan/dev/github/go/snivilised/extendio/xfs/nav/traverse-navigator_test.go:196
------------------------------
• [0.001 seconds]
TraverseNavigator errors Extension error should: not panic
/home/plastikfan/dev/github/go/snivilised/extendio/xfs/nav/traverse-navigator-error_test.go:198

  Timeline >>
  ---> 🧲 [traverse-navigator-test:BEGIN], root: '/home/plastikfan/dev/github/go/snivilised/extendio/Test/data/MUSICO/RETRO-WAVE'
  ===> path:'/home/plastikfan/dev/github/go/snivilised/extendio/Test/data/MUSICO/RETRO-WAVE/Chromatics/Night Drive/A1 - The Telephone Call.flac'
  ===> path:'/home/plastikfan/dev/github/go/snivilised/extendio/Test/data/MUSICO/RETRO-WAVE/Chromatics/Night Drive/A2 - Night Drive.flac'
  ===> path:'/home/plastikfan/dev/github/go/snivilised/extendio/Test/data/MUSICO/RETRO-WAVE/College/Northern Council/A1 - Incident.flac'
  ===> path:'/home/plastikfan/dev/github/go/snivilised/extendio/Test/data/MUSICO/RETRO-WAVE/College/Northern Council/A2 - The Zemlya Expedition.flac'
  ===> path:'/home/plastikfan/dev/github/go/snivilised/extendio/Test/data/MUSICO/RETRO-WAVE/College/Teenage Color/A1 - Can You Kiss Me First.flac'
  ===> path:'/home/plastikfan/dev/github/go/snivilised/extendio/Test/data/MUSICO/RETRO-WAVE/College/Teenage Color/A2 - Teenage Color.flac'
  ===> path:'/home/plastikfan/dev/github/go/snivilised/extendio/Test/data/MUSICO/RETRO-WAVE/Electric Youth/Innerworld/A1 - Before Life.flac'
  ===> path:'/home/plastikfan/dev/github/go/snivilised/extendio/Test/data/MUSICO/RETRO-WAVE/Electric Youth/Innerworld/A2 - Runaway.flac'
  << Timeline
------------------------------
S [SKIPPED]
TraverseNavigator Folders With Files 🧪 ===> given: 'folders(with files): Path is leaf'
/home/plastikfan/dev/github/go/snivilised/extendio/xfs/nav/traverse-navigator_test.go:263
------------------------------
S [SKIPPED]
TraverseNavigator Folders With Files 🧪 ===> given: 'folders(with files): Path contains folders (check all invoked)'
/home/plastikfan/dev/github/go/snivilised/extendio/xfs/nav/traverse-navigator_test.go:274
------------------------------

Ran 1 of 81 Specs in 0.018 seconds
SUCCESS! -- 1 Passed | 0 Failed | 0 Pending | 80 Skipped
```

I'm guessing that since there appears to big changes since version 2.3, that something significant has changed in this area and I am not doing something right. I've looked in the docs and can't find anything that relates to this (that doenst mean its not there, its just that I havent found it!)

Is there a way of silencing the output of skipped tests when running in focused mode? Essentially I want it to be verbose for the focused test and silent for the skipped ones.

Thanks.

**Comment by onsi** [comment]
hey! i recently added the notion a unified timeline to ginkgo's output and in doing so tweaked how verbosity affects output.  i agree this is too noisy.  let me tweak it some more and cut a new release...

**Comment by plastikfan** [comment]
Ok thanks @onsi

**Comment by onsi** [comment]
this is now fixed on Ginkgo 2.5.1

**Comment by plastikfan** [comment]
Super duper @onis!
